src/makeCalculator.js

```diff
@@ -4,7 +4,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function (valueResult) {
+      return function (value) {
+        return valueResult + value;
+      };
+    },
+
+    subtract: function (valueResult) {
+      return function (value) {
+        return valueResult - value;
+      };
+    },
+
+    multiply: function (valueResult) {
+      return function (value) {
+        return valueResult * value;
+      };
+    },
+
+    divide: function (valueResult) {
+      return function (value) {
+        if (value > 0) {
+          return valueResult / value;
+        } else {
+          return 'Error';
+        }
+      };
```

Comment:
Use method shorthand for the `divide` function. Also, instead of returning 'Error' as a string, consider throwing an error or handling the division by zero case differently, as returning a string is not consistent with the other methods that return numbers.